source/administration/config-settings.rst

```diff
@@ -2273,6 +2302,15 @@ There are a number of settings customizable in ``config.json`` unavailable in th
 Service Settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
+License File Location
+^^^^^^^^^^^^^^^^^^^^^
+
+Path and filename of the license file on disk. On startup, if Mattermost cannot find a valid license in the file ``config/mattermost.mattermost-license``, it looks here. It can be an absolute path, or a path relative to the ``mattermost`` directory.
```

Comment:
If mattermost cannot find an existing license file in the database from a previous upload, then it looks at LicenseFileLocation as a fallback.